python/cog/server/http.py

```diff
@@ -104,6 +105,7 @@ def create_app(
     threads: int = 1,
     upload_url: Optional[str] = None,
     mode: str = "predict",
+    is_build: bool = False
```

Comment:
I like the decision to define this behavior in a new argument to `create_app`. We originally discussed defining a new `mode` value, but I see the value of using something that doesn't map to a command-line argument, and therefore risk misuse (e.g. What's the expected behavior of a pod running a model with the build mode?)

Reply:
I'm not wild about this particular spelling, but I can't think of anything better. Since it's effectively a private API, it's probably fine. Or else, we can change it later without breaking anyone.